app/helpers/application_helper.rb

```diff
@@ -343,12 +343,27 @@ def name_to_id(name)
     name.to_s.gsub(/\]\[|[^-a-zA-Z0-9:.]/, "_").sub(/_$/, "")
   end
 
-  def field_id(form, attribute)
-    name_to_id(field_name(form, attribute))
+  def field_id(form_or_object_name, attribute, index: nil, **_kwargs)
+    name_to_id(field_name(form_or_object_name, attribute, index: index))
   end
 
-  def field_name(form, attribute)
-    "#{form.object_name}[#{field_attribute(attribute)}]"
+  # This is a partial re-implementation of ActionView::Helpers::FormTagHelper#field_name.
+  # The method contract changed in Rails 7.0, but we can't use the default because it sometimes
+  # includes other information that -- at a minimum -- wrecks havoc on the Cucumber feature tests.
```

Comment:
"wreaks havoc" rather than "wrecks havoc," I believe? (I had to look it up because I knew it _sounded_ like "reek," but wasn't spelled that way. 😆)

Reply:
Thanks! I have no idea why I did that, I know it's not spelled that way :facepalm: